worldmap.js

```diff
@@ -117,11 +116,95 @@ module.exports = function(RED) {
             }
             node.status({});
         });
-        sockets[this.path].on('connection', callback);
+        sockets[node.path].on('connection', callback);
+    }
+
+    var WorldMap = function(n) {
+        worldMap(this, n);
     }
     RED.nodes.registerType("worldmap",WorldMap);
 
 
+    function HTML(ui, config) {
+        var width = config.width;
+        if (width === 0) {
```

Comment:
actually these need to be == 0 as when set to auto the numbers come through as strings.

Reply:
Thank you! fixed.